src/main/java/pl/edu/agh/ztis/newsanalyzer/NewsAnalyzerApplication.java

```diff
@@ -18,23 +19,31 @@
     @Autowired
     private FeedRepository repository;
 
+    private String filepath;
+    private Map<String, String> dict;
+
 	public static void main(String[] args) {
 		SpringApplication.run(NewsAnalyzerApplication.class, args);
 	}
 
 	@Override
 	public void run(String... args) throws Exception {
+	    filepath = "./src/main/resources/result_dict_v1-lowercase.csv";
+        DictReader dictReader = new DictReader(filepath);
+        dict = dictReader.read();
+
 
         List<Extractor> extractors = Arrays.asList(
-                new BBCExtractor("http://feeds.bbci.co.uk/news/rss.xml", repository),
-                new WSJExtractor("http://www.wsj.com/xml/rss/3_7085.xml", repository),
-                new TVNExtractor("https://www.tvn24.pl/najwazniejsze.xml", repository));
+                new BBCExtractor("http://feeds.bbci.co.uk/news/rss.xml", repository, dict),
```

Comment:
Maybe we could extract this dict somehow? There will be many more extractors added and then we have to add it as parameter to every one of them.